docs/existing-project/update-atri.md

```diff
@@ -55,6 +55,9 @@ pip install atri -U
 
 Currently, the below process is manual, and soon these steps will be automated.
 
+<Tabs>
+<TabItem value="pipenv" label="Mac/Linux" default>
+
 ```shell
 cd project_dir
```

Comment:
Replace `project_dir` with `my_app` since this is the name of the directory in this example.